wzdx/raw_to_standard/planned_events.py

```diff
@@ -8,75 +8,102 @@
 import uuid
 from collections import OrderedDict
 
-from wzdx.tools import date_tools, polygon_tools, wzdx_translator
-from wzdx.util.collections import PathDict
-
-PROGRAM_NAME = 'PlannedEventsRawToStandard'
-PROGRAM_VERSION = '1.0'
-
-STRING_DIRECTION_MAP = {'north': 'northbound', 'south': 'southbound',
-                        'west': 'westbound', 'east': 'eastbound'}
+import pytz
+import regex
+
+from ..tools import (
+    cdot_geospatial_api,
+    date_tools,
+    geospatial_tools,
+    polygon_tools,
+    wzdx_translator,
+    combination,
+)
+from ..util.collections import PathDict
+
+PROGRAM_NAME = "PlannedEventsRawToStandard"
+PROGRAM_VERSION = "1.0"
+
+STRING_DIRECTION_MAP = {
+    "north": "northbound",
+    "south": "southbound",
+    "west": "westbound",
+    "east": "eastbound",
+}
 
-REVERSED_DIRECTION_MAP = {'northbound': 'southbound', 'southbound': 'northbound',
-                          'eastbound': 'westbound', 'westbound': 'eastbound'}
+REVERSED_DIRECTION_MAP = {
+    "northbound": "southbound",
+    "southbound": "northbound",
+    "eastbound": "westbound",
+    "westbound": "eastbound",
+}
 
 WORK_ZONE_INCIDENT_TYPES = {
-    "Maintenance Operations":   {"Traffic": True},
-    "Other":                    {"Environmental": True},
-    "Emergency Roadwork":       {"Traffic": True},
+    "Maintenance Operations": {"Traffic": True},
+    "Emergency Roadwork": {"Traffic": True},
 }
 INCIDENT_ID_REGEX = "^OpenTMS-Incident"
 
 
 def main():
+    cdotGeospatialApi: cdot_geospatial_api.GeospatialApi = (
+        cdot_geospatial_api.GeospatialApi()
+    )
     source_file, output_dir = parse_rtdh_arguments()
-    input_file_contents = open(source_file, 'r').read()
+    input_file_contents = open(source_file, "r").read()
     generated_messages = generate_standard_messages_from_string(
-        input_file_contents)
+        cdotGeospatialApi, input_file_contents
+    )
 
     generated_files_list = []
     for message in generated_messages:
         output_path = f"{output_dir}/standard_planned_event_{message['event']['source']['id']}.json"
-        open(output_path, 'w+').write(json.dumps(message, indent=2))
+        open(output_path, "w+").write(json.dumps(message, indent=2))
         generated_files_list.append(output_path)
 
     if generated_files_list:
-        print(
-            f"Successfully generated standard message files: {generated_files_list}")
+        print(f"Successfully generated standard message files: {generated_files_list}")
```

Comment:
Print to logging swap

Reply:
Updated other logging statements to use print